docs/root/version_history/current.rst

```diff
@@ -97,6 +97,7 @@ New Features
 * listener: new listener metric ``downstream_cx_transport_socket_connect_timeout`` to track transport socket timeouts.
 * rbac: added :ref:`destination_port_range <envoy_v3_api_field_config.rbac.v3.Permission.destination_port_range>` for matching range of destination ports.
 * route config: added :ref:`dynamic_metadata <envoy_v3_api_field_config.route.v3.RouteMatch.dynamic_metadata>` for routing based on dynamic metadata.
+* router: added timeseries metrics for remaining error cases for internal redirect failures.
```

Comment:
Can you put the specific error cases here( e.g. recreate stream failed) instead of "remaining error cases".

Reply:
done!